Linter is broken with stylint >1.4.1

```
Error: module.js:340 throw err; ^

Error: Cannot find module '//package.json' at Function.Module._resolveFilename (module.js:338:15) at Function.Module._load (module.js:289:25) at Module.require (module.js:366:17) at require (module.js:385:17) at Object. (/usr/local/lib/node_modules/stylint/index.js:31:14) at /usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:91:25 at forEach (/usr/local/lib/node_modules/stylint/node_modules/mout/array/forEach.js:15:27) at Function.factory [as create] (/usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:89:7) at Object. (/usr/local/lib/node_modules/stylint/bin/stylint:42:11) at Module._compile (module.js:425:26)
Error: module.js:340
    throw err;
    ^
Error: Cannot find module '//package.json'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/stylint/index.js:31:14)
    at /usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:91:25
    at forEach (/usr/local/lib/node_modules/stylint/node_modules/mout/array/forEach.js:15:27)
    at Function.factory [as create] (/usr/local/lib/node_modules/stylint/node_modules/stampit/stampit.js:89:7)
    at Object.<anonymous> (/usr/local/lib/node_modules/stylint/bin/stylint:42:11)
    at Module._compile (module.js:425:26)
    at parameters.exit (/Users/Zeno/.atom/packages/linter-jshint/node_modules/atom-linter/lib/helpers.js:47:27)
    at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:215:47)
    at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:237:18)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
```

--- a/lib/init.coffee
+++ b/lib/init.coffee
@@ -42,7 +42,9 @@ module.exports =
   provideLinter: ->
     helpers = require('atom-linter')
     provider =
-      grammarScopes: ['source.stylus', 'source.styl', 'source.css.styl', 'source.css.stylus']
+      grammarScopes: [
+        'source.stylus', 'source.styl', 'source.css.styl', 'source.css.stylus'
+      ]
       scope: 'file'
       lintOnFly: true
 
@@ -55,16 +57,29 @@ module.exports =
 
         projectConfigPath = helpers.find(filePath, @projectConfigFile)
 
+        # Attempt to use Atom's project folder for the CWD
+        if !projectConfigPath
+          projectDir = atom.project.relativizePath(filePath)[0]
+
+        # Fall back to the file directory if Atom wasn't opened as a project
+        if !projectDir
+          projectDir = path.dirname(filePath)
+
         parameters = [filePath]
 
         if(@onlyRunWhenConfig && !projectConfigPath)
-          atom.notifications.addError 'Stylint config no found'
+          atom.notifications.addWarning 'Stylint config not found'
           return Promise.resolve([])
 
         if(@onlyRunWhenConfig || !@runWithStrictMode && projectConfigPath)
           parameters.push('-c', projectConfigPath)
 
-        return helpers.execNode(@executablePath, parameters, stdin: fileText).then (result) ->
+        options = {
+          stdin: fileText
+          cwd: projectDir
+        }
+
+        return helpers.execNode(@executablePath, parameters, options).then (result) ->
           regex = /(Warning|Error):\s(.*)\nFile:\s(.*)\nLine:\s(\d*)/g
           messages = []
 

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
   "dependencies": {
     "atom-linter": "^4.0.0",
     "atom-package-deps": "^4.0.1",
-    "stylint": "1.4.1"
+    "stylint": "1.3.10"
   },
   "package-deps": [
     "linter"

